### PR TITLE
Update getting started guide for non-developers

### PR DESCRIPTION
**Pull request by settermjd**
As requested in #523, this PR updates the getting started guide with a section that helps non-developers get started contributing to the documentation. It steps users through the absolute minimum required to get started.

This is currently a work in progress, as I'm very keen to have it be something that is as relatable to non-developers as possible.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -16,7 +110,7 @@ To contribute to the documentation, you need to:
 7. Contributing to the documentation
 8. Getting support
 
-## Setup Your Local Copy of the Docs Repository
+### Setup Your Local Copy of the Docs Repository
 
 With this done, you then need to get your local copy of the docs repository ready.
 To do this, follow the following three steps.
```
Comment (pako81) [inline docs/getting-started.md]: change the wording?

File: docs/getting-started.md [hunk 2]
```diff
@@ -2,7 +2,101 @@
...
+* item 1
+* item 2
+
+[source,ruby]
+====
+puts "Hello, World!"
+====
+```
+
+Have a read through [the AsciiDoctor Syntax Quick Reference](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/) to familiarise yourself with the format's essentials.
```
Comment (mmattel) [inline docs/getting-started.md]: You may want to add:
`Please also refer to [Best Practices and Tips](./docs/best-practices.md) for more information.`

File: docs/getting-started.md [hunk 3]
```diff
@@ -2,7 +2,101 @@
...
+All that you need is a web browser, such as [Google Chrome](https://www.google.com/chrome/index.html), [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/new/), [Apple Safari](https://www.apple.com/safari/), or [Microsoft Edge](https://www.microsoft.com/en-us/windows/microsoft-edge).
+
+### Initial Steps
+
+Before you can contribute to the documentation, you need to:
+
+1. [Have a GitHub account](#have-a-github-account).
+2. [Know the AsciiDoc basics.](#asciidoc-basics)
+
+<a name="have-a-github-account"></a>
```
Comment (tboerger) [inline docs/getting-started.md]: Why are you adding these links? GitHub automatically links all headlines.

Comment (settermjd) [reply docs/getting-started.md]: I wasn't aware that it did. I didn't see anything in their guide.

Comment (settermjd) [reply docs/getting-started.md]: On quick review, I see/remember that you're correct. I'll make a subsequent PR.

**Pull request merged**